index.js

```diff
@@ -1,184 +1,249 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
-
-
+// console.log("%cQuestion 1", "color: red")
+ 
+let counter = 1
+counter = 2
+console.log(counter)
+// console.log("%c----------", "color: red")
+ 
+ 
+ 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
+// *** after testing, comment the line causing the error back in
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
-
-
-
+// console.log("%cQuestion 2", "color: red")
+ 
+const name = "Sam"
+ 
+ 
+ 
```

Comment:
Nice work using the "let" and "const" keywords in the correct context when defining JS variables.